java-apps/java-rabbitmq-entry-point/src/main/java/rabbitmqentrypoint/producer/Producer.java

```diff
@@ -0,0 +1,25 @@
+package rabbitmqentrypoint.producer;
+
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import rabbitmqentrypoint.properites.RelatedServicesProperties;
+
+@Component
+public class Producer {
+
+    private final RabbitTemplate rabbitTemplate;
+
+    @Autowired
+    private RelatedServicesProperties relatedServicesProperties;
+
+    public Producer(RabbitTemplate rabbitTemplate) {
+        this.rabbitTemplate = rabbitTemplate;
+    }
+
+    public void send(byte[] message) throws Exception {
+        System.out.println("Sending message...");
```

Comment:
Maybe this should print what message is being sent?

```suggestion
        System.out.println("Sending message... <" + new String(message) + ">");
```